src/js/components/PeaksList/Peak.module.css

```diff
@@ -0,0 +1,39 @@
+.card{
+    display:grid;
+    grid-template-rows:40vh auto;
+    overflow:hidden;
+    position:relative;
+    box-shadow: 8px 4px 16px -2px rgba(90,70,69,0.5);
+    background-color:rgb(238, 235, 235);
+    border-radius:20px;
+    margin-top:5%;
+    margin-bottom: 5%;
+}
+
+.photo{
+    position: relative;
+    width:100%;
+    height:100%;
+    object-fit:cover;
+    object-position: center;
+}
+
+.card:hover .showMoreButton{
+    display:block;
+}
+
```

Comment:
Jak zmienisz strukturę komponentów z linkiem to można to ograć dużo łatwiej 
za pomocą css filter, wtedy showMoreButton jest niepotrzebny

Reply:
Usunęłam showMoreButton i zostawiłam card:hover